app/javascript/src/components/tenting/Step1.jsx

```diff
@@ -0,0 +1,77 @@
+import React, { Component } from 'react';
+
+// semantic ui components
+import { Menu, Image, Button, Form, Step, Divider, Message, Header, Segment, Card, Grid, Table, Icon } from 'semantic-ui-react';
+
+// logos
+import * as kvilleLogo from './../../images/kville.png';
+// import FAQ page
+import FAQ from './FAQ';
+
+class Step1 extends Component {
+  render () {
+    const square = { width: 175, height: 175 };
+    return (
+      <div className="body">
+       <Image centered src={kvilleLogo} size='massive'/>
+       <Message centered header='Step 1' size='large' content='Find your tent members and establish a Tent Captain.'/>
```

Comment:
`Message` does not have a centered prop. Please check docs to confirm props before using them as they throw errors or warnings (check your console, and you can see them).